src/NServiceBus.Core/Pipeline/Incoming/SatelliteTransportReceiveToPhysicalMessageProcessingConnector.cs

```diff
@@ -0,0 +1,15 @@
+namespace NServiceBus
+{
+    using System;
+    using System.Threading.Tasks;
+    using Pipeline;
+
+    class SatelliteTransportReceiveToPhysicalMessageProcessingConnector : StageForkConnector<ITransportReceiveContext, IIncomingPhysicalMessageContext, IBatchDispatchContext>
```

Comment:
Why is this a `StageForkConnector`? I think it should be a `StageConnector<ITransportReceiveContext, IIncomingPhysicalMessageContext>` only, unless you intend to support batching (which I think is overkill for satellites)